index.html

```diff
@@ -270,6 +280,11 @@ <h1 class="about-text">About Myself</h1>
                   />
                 </a>
               </li>
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you make the `horizontal overflow` to be `hidden` as right it makes your design looks messy, it also add an extra spacing when we scroll it. And this will ensure that your design sticks as much as possible to the one of your Figma chosen template.
![Screenshot from 2023-06-08 18-14-18](https://github.com/JavierAybar/Portfolio/assets/83042742/92476913-a63a-4517-bfcb-3795a82503c6)